news/2017-12-20-owc-2017-concludes.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "osu! World Cup 2017 Concludes"
+date: 2017-12-20 10:00:00 +0000
+---
+
+Last weekend, the final circles were clicked, and the victors decided. 36 teams from across the globe whittled down to only 6, and from them, only 1 emerged victorious. After a hard-fought battle, **Poland** emerged victorious as the osu! Champions of 2017 - read on to find out more!
+
+![](/wiki/shared/news/banners/owc_2017_banner.jpg)
+
+This year, 36 teams from across the globe entered and competed in the biggest circle-clicking spectacle that osu! has to offer. From new teams that just entered the fray like Romania to older returning ones like Japan, the tournament saw an overall increase in competitiveness from past years - joined with fiendishly difficult mappools to test skills of some of osu!'s best players.
+
+The Finals Weekend saw 6 remaining teams – USA, Poland, UK, South Korea, Taiwan, and Germany – competing for the ultimate title of **osu! Champion**.
+
+In a decisive display of hard-won skill, **Poland** reigned supreme in the final match, taking home the osu! World Cup and the title until next year. If you supported them with a spiffy profile banner, you get to keep yours and broadcast your choice proudly till the next tournament starts! 
```

Comment:
Would it be better to say the next "official" tournament. Well... assuming MWC7K even happens lol